src/command-registry.coffee

```diff
@@ -18,6 +18,11 @@ SequenceCount = 0
 # command event listeners globally on `atom.commands` and constrain them to
 # specific kinds of elements with CSS selectors.
 #
+# Command names must follow the `namespace:event` pattern, where both parts can
+# be further split into words by hyphens. Typically `namespace` will be the
```

Comment:
98% of the time that's likely fine. But only a few lines further down there's a reference to a user-defined command example `user:insert-date`, so I didn't want to be overly restrictive here, since there are valid scenarios where it's **not** tied to a package. Hmm...